forum/qa-plugin/outdated-question-info/frontend/show-outdated-question-info.js

```diff
@@ -7,23 +7,28 @@ if(placeOfOutdatedQuestionInfo){
     const publishMonthNewerThanNow = publishDate.getMonth() - 1 >= now.getMonth();
 
     if (publishYearOlderThanNow && publishMonthNewerThanNow) {
-        let infoDisplay = "";
-        if(placeOfOutdatedQuestionInfo.style.display !== 'none'){
-            infoDisplay = "display:block;";
-        }else{
-            infoDisplay = "display:none;";
+        let shouldBeDisplayed = "";
+        if(placeOfOutdatedQuestionInfo.style.display === 'none'){
+            shouldBeDisplayed = "hidden";
         }
         placeOfOutdatedQuestionInfo.insertAdjacentHTML('beforebegin', 
-                `<p class = "qa-outdated-question-container" style = "${infoDisplay}">
+                `<p class = "qa-outdated-question-container ${shouldBeDisplayed}">
                     To pytanie zostało zadane już dawno temu i może być nieaktualne.<br/>
                     Upewnij się, że Twoja odpowiedź nadal będzie pomocna.
                 </p>`);
     }
     const QuestionElemExist = document.querySelector('.qa-outdated-question-container');
 
     if(QuestionElemExist){
+        const cancelAnswer = document.querySelectorAll('input[name=docancel]')[1];
```

Comment:
Z tego co widzę w DOM inspektorze na forum, to inputy `[name=docancel]` służą za czerwony przycisk anulujący pisanie komentarza (lub odpowiedzi, jeśli jest już jakaś udzielona). Jeśli więc jest potrzeba złapania przycisku anulującego aktualnie pisany post, to raczej na pewno nie można tego tak zahardkodować.

Można wyciągnąć wszystkie te inputy (przyciski) i w pętli podpiąć im event listener, albo podpiąć się pod [aktualnie aktywną instancję CKEditora](https://ckeditor.com/docs/ckeditor4/latest/api/CKEDITOR.html#property-currentInstance) i relatywnie od niej znaleźć przycisk anulujący; jakbyś miał z tym problem, to daj znać, pomogę (w trakcie developmentu dla ułatwienia można sobie wrzucić `console.log(CKEDITOR.currentInstance)` w paro sekundowy `setTimeout`, żeby móc zfocusować edytor na czas sprawdzania instancji - inaczej to property zwróci `null`). Ewentualnie zmatchować najbliższego przodka edytora i przycisku anulującego na podstawie klucza z obiektu [`CKEDITOR.instances`](https://ckeditor.com/docs/ckeditor4/latest/api/CKEDITOR.html#property-instances).